### PR TITLE
Fixes in menu

### PR DESCRIPTION
**Pull request by annakocheshkova**
- Moved Code Push to the top-level
- Moved Link Code Push under Code Push
- Added a setting to show/hide status bar two commands for it. Also added Hide status bar item under Settings.

### CODE REVIEW

File: src/appcenter/commands/codepush/index.ts [hunk 1]
```diff
@@ -2,4 +2,5 @@ export { default as SetCurrentDeployment } from './setCurrentDeployment';
 export { default as ReleaseReact } from './releaseReact';
 export { default as SwitchMandatoryPropForRelease } from './switchMandatoryPropForRelease';
 export { default as SetTargetBinaryVersion } from './setTargetBinaryVersion';
+export { default as LinkCodePush } from './linkCodePush';
```
Comment (ruslan-bikkinin) [inline src/appcenter/commands/codepush/index.ts]: Does it necessary? I mean this is the only change in the file.

Comment (ruslan-bikkinin) [reply src/appcenter/commands/codepush/index.ts]: oh, nevermind

File: src/commandHandlers/settingsCommandHandler.ts [hunk 2]
```diff
@@ -29,4 +32,20 @@ export default class SettingsCommandHandler extends BaseCommandHandler {
     public async SwitchVstsAcc(): Promise<void> {
         await new Settings.SwitchVstsAccount(this.getCommandParams()).runNoClient();
     }
+
+    public async ShowStatusBar(): Promise<void> {
```
Comment (max-mironov) [inline src/commandHandlers/settingsCommandHandler.ts]: Id prefer to name it `SetShowStatusBar` as this is not actually Showing it but just changing setting value
The same is for Hide.

File: src/strings.ts [hunk 3]
```diff
@@ -112,6 +114,10 @@ export class Strings {
     public static ProvideVSTSProjectPromptMsg: string = "Please specify VSTS Project";
     public static FetchDeploymentsStatusBarMessage: string = "Fetching CodePush deployments for you...";
     public static InvalidCurrentAppNameMsg: string = "Sorry, the app name isn't valid.";
+    public static StatusBarAlreadyShown: string = "The status bar is already shown. Look, it's in the bottom left corner of the window, with your name/app name on it!";
```
Comment (max-mironov) [inline src/strings.ts]: Not sure if this message is ok, but we can change it later if needed :)

**Pull request merged**